src/components/Layout/Layout.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
You can do this `import React, {Fragment} from 'react'` and then use `<Fragment>` instead of `<React.Fragment>` or you can just use `<>` without any import

Reply:
This is done